src/java/org/apache/cassandra/io/sstable/IndexInfo.java

```diff
@@ -47,7 +47,7 @@
  * (long) IndexInfo.offset
  * (long) IndexInfo.width
  * (bool) IndexInfo.endOpenMarker != null              (if 3.0)
- *  (int) IndexInfo.endOpenMarker.localDeletionTime    (if 3.0 && IndexInfo.endOpenMarker != null)
+ * (long) IndexInfo.endOpenMarker.localDeletionTime    (if 3.0 && IndexInfo.endOpenMarker != null)
```

Comment:
This refers to serialization, and `DeletionTime#localDeletionTime` is still being serialized as a `int`.

Reply:
I changed it to `(int (uint post c14427))` wdyt?

Reply:
Looks perfect to me.